client/my-sites/plans/current-plan/header.jsx

```diff
@@ -67,7 +67,7 @@ class CurrentPlanHeader extends Component {
 				<div className={ classes }>
 					<span className="current-plan__header-expires-in">
 						{ hasAutoRenew
-							? translate( 'Set to Auto Renew on %s.', { args: currentPlan.userFacingExpiryMoment.format( 'LL' ) } )
+							? translate( 'Set to Auto Renew on %s.', { args: currentPlan.autoRenewDateMoment.format( 'LL' ) } )
 							: translate( 'Expires on %s.', { args: currentPlan.userFacingExpiryMoment.format( 'LL' ) } )
```

Comment:
> Hopefully no one will confuse its name ("user facing expiry") with meaning that it's the expiry date user should see. Because it's not. 

It is. It is we ( and to be honest - me ) that confused EXPIRY with RENEWAL which is the main reason for this error. The date is when the plan expires but we renew it a month earlier to have the time for retry, etc. So the prop is 100% aptly named, we just used it to show something different :)